.circleci/config.yml

```diff
@@ -14,26 +20,39 @@ parameters:
     type: string
     default: "echo \"no release-command specified\""
 
+###
+# Executors
+###
 executors:
   default:
     docker:
-      - image: cimg/node:lts
+      - image: cimg/node:18.19
```

Comment:
Why lock it to a specific version? We don't want to always test on latest?

Reply:
Good question. I wasn't sure if we wanted to do something like "default" executor and "canary" executor and run the jobs twice, one on an older node version that we claim support for and one for newer versions so we can see errors coming. Do you want to revert this and just use latest?

Reply:
Ideally, we want something that runs on both latest lts and a version that we know always works. Not sure how easy that is, and it might be out of scope. We can create a jira and do that later if it is a lot of effort.

Reply:
OK, I will create a ticket to have a canary build. For now, is it OK to just glue to 18?

Reply:
yes, 18 is fine

Reply:
Here's the ticket: https://issues.corp.twilio.com/browse/VBLOCKS-2619